### ISSUE TITLE
Something wrong with reading from source once split happens

### ISSUE THREAD

**Issue by zenjabba** [ivandeex/rclone#21]
2019/09/07 00:56:22 ERROR : Angels & Demons (2009)/Angels.and.Demons.2009.Theatrical.Cut.2160p.UHD.BluRay.REMUX.HDR.HEVC.Atmos-EPSiLON.mkv: Failed to copy: multipart upload failed to read source: file already closed

When copying from google drive to box (1GB Split) once it gets ready to do it's split it gets this error.

**Comment by zenjabba** [comment]
/usr/bin/rclone sync dm-4k-movies:/ box_chunker:/source/4k-movies -v --transfers=1 --checkers=20

**Comment by zenjabba** [comment]
2019/09/07 01:05:13 DEBUG : Arrival (2016)/Arrival.2016.UHD.BluRay.2160p.DTS-HD.MA.7.1.HEVC.REMUX-FraMeSToR.mkv__tmp.1567818208.000: Uploading part 32/32 offset 992M/1G part size 32M

2019/09/07 01:05:17 DEBUG : Arrival (2016)/Arrival.2016.UHD.BluRay.2160p.DTS-HD.MA.7.1.HEVC.REMUX-FraMeSToR.mkv__tmp.1567818208.001: Cancelling multipart upload: multipart upload failed to read source: EOF 

2019/09/07 01:05:19 DEBUG : Arrival (2016)/Arrival.2016.UHD.BluRay.2160p.DTS-HD.MA.7.1.HEVC.REMUX-FraMeSToR.mkv: Received error: multipart upload failed to read source: EOF - low level retry 1/10

**Comment by zenjabba** [comment]
[box_chunker]
type = chunker
remote = box:/
chunk_size = 1G

**Comment by ivandeex** [comment]
This beta does not support multipart uploads yet.
Please set multipart cutoff equal to chunk size. This effectively disables multipart upload of chunks.

**Comment by zenjabba** [comment]
Would you mind telling me how to do that

**Comment by ivandeex** [comment]
https://rclone.org/box/#box-upload-cutoff
Either on command line
```
rclone ...... --box-upload-cutoff=1G .....
```
or modify rclone.conf section
```
[box_chunker]
type = chunker
remote = box:/
chunk_size = 1G
upload_cutoff = 1G
```
Note: We are planning to change Chunker metadata format before merging into rclone v1.50. You'll have to re-upload your files after that. Please don't use this fork on critical data. You are warned.

**Comment by ivandeex** [comment]
@zenjabba Please try the latest beta

**Comment by ivandeex** [comment]
Issues with multi-uploads are fixed in current beta, the fix will be published as part of rclone v1.50 this October.
Closing due to no reply from user.

**Comment by zenjabba** [comment]
Sorry, that fixed it. I generally work on this stuff on the weekend
